pkg/install/install.go

```diff
@@ -253,6 +257,21 @@ func Install(ctx context.Context, image string, opts *Options) error {
 	return nil
 }
 
+func validateMemoryArgs(defaultMemory int64, maximumMemory int64) error {
+
+	//if default is set to infinite memory (0) max memory must also be set to (0)
+	if defaultMemory == 0 && maximumMemory != 0 {
+		return fmt.Errorf("invalid memory args: workload-memory-default set to unlimited which exceeds the workload-memory-maximum of %v",
+			maximumMemory)
+	}
```

Comment:
Right now in the `run` and `Acornfile` if you specify `0` for their memory when the max is not `0`, they will instead be given the maximum memory. I'd say we should also follow that behavior here so that a user can say "here is my maximum memory and for the default I want the workloads to get as much as they can". 

Even better, in a default installation a user can specify just the `--workload-memory-maximum` flag and all workloads will default to that maximum since the `--workload-memory-default` flag is `.`. What if we instead added some output here to just let a user know what's happening?

Reply:
so if max memory != 0 and default = 0 we currently set default = max?

Reply:
In effect, that logic happens here:

https://github.com/acorn-io/acorn/blob/6c93a9ed787abe927b3928f7db6895832e26924d/pkg/apis/internal.acorn.io/v1/memory.go#L115-L118

Reply:
So if we get to that line and the amount of bytes that we would set would be equal to zero (say in the case that we're using the default) then we will get the maximum amount of memory instead.

Reply:
pushed up a change let me know if that seems better ux

Reply:
Two things here. 

1. Right now testing locally I realized that we're outputting the full byte amount to the user in the error message. Could you update this to parse it and then turn it into a suffixed string like we do elsewhere in the code base?
    ```
    ✗  ERROR:  invalid memory args: workload-memory-default set to 1048576000 which exceeds the workload-memory-maximum of 524288000
    ```

2. The warning right now is a bit misleading - there's nothing actually wrong, just a bit magical so communicating it is nice. Can you update this to use `pterm.Info.Println()` and remove the warning part? I'm envisioning this to be a similar UX to how LE communicates info [here](https://github.com/acorn-io/acorn/blob/6fa2978f51e27e98dd14dd6c98e3e838cd649640/pkg/install/install.go#L135).

Reply:
Nice, most recent push looks good.